modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala

```diff
@@ -125,6 +128,31 @@ object SbtAlg {
       val sbtDir: F[File] =
         fileAlg.home.map(_ / ".sbt")
 
+      val buildPropertiesFile: F[File] =
+        fileAlg.home.map(_ / "project" / "build.properties")
+
+      final private val sbtVersionRegex = s"build.properties=(.+)".r
+      final private val latestSbtVersions = List(sbt.defaultSbtVersion, sbt.latestSbtVersion_0_13)
+        .map(_.value)
+
+      def proposeBuildPropertiesUpdate(): F[List[Update.Single]] =
+        buildPropertiesFile.flatMap { prop =>
+          fileAlg
+            .readFile(prop)
+            .map { maybeContent =>
+              for {
+                content <- maybeContent
+                currentVer <- sbtVersionRegex.findFirstMatchIn(content).map(_.group(1))
+                if !latestSbtVersions.contains(currentVer)
+                newVer = if (currentVer.startsWith("0."))
```

Comment:
We need to ensure that `newVersion` is actually greater than `currentVer`. `model.Version` can be used to do that.

Reply:
Done in https://github.com/fthomas/scala-steward/pull/665/commits/1578a3dbe3128cdae517c687a5b85aa5c6e2b3cd